src/main/kotlin/Config.kt

```diff
@@ -84,10 +84,7 @@ object Config {
         var topicRegex: Pattern = Pattern.compile(getEnv("K2HB_KAFKA_TOPIC_REGEX") ?: "db.*")
         var dlqTopic = getEnv("K2HB_KAFKA_DLQ_TOPIC") ?: "dead-letter-queue"
 
-        fun reportTopicSubscriptionDetails(): String {
```

Comment:
I think we should keep some of this?

Reply:
Everything that it returned is logged in subsequent messages. Previously it was all logged twice.